include/quic/address.hpp

```diff
@@ -261,7 +293,7 @@ namespace oxen::quic
         ngtcp2_path _path{local, remote, nullptr};
 
       public:
-        Path() = default;
+        // Path() = default;
```

Comment:
Is making this non-default-constructible intentional?  Without this, `Path` can end up being a bit annoying to work with (i.e. I can't do something like: `Path p; if (a) p = x; else p = y;`), but perhaps that's okay with how we are using it?

Reply:
Mistake, thank you